packages/config/index.js

```diff
@@ -170,6 +185,45 @@ BemConfig.prototype.levelMap = function() {
     });
 };
 
+BemConfig.prototype.levels = function(setName) {
+    var _this = this;
+
+    return this.get().then(function(config) {
+        var levels = config.levels || [],
+            sets = config.sets || {};
+
+        if (!sets[setName]) { return []; }
+
+        var resolvedSets = resolveSets(sets),
+            set = resolvedSets[setName];
+
+        if (!set || !set.length) { return []; }
+
+        // TODO: uniq
+        return Promise.all(set.map(setDescription => {
+            if (setDescription.library) {
+                return _this.library(setDescription.library).then(libConfig => {
+                    assert(libConfig, 'Library `' + setDescription.library + '` was not found');
+
+                    return libConfig.levels(setDescription.set);
+                });
+            }
+
+            if (setDescription.set) {
+                return _this.levels(setDescription.set);
+            }
+
+            var level = levels.find(lvl => {
+                return lvl.layer === setDescription.layer;
```

Comment:
should we check `setDescription` has `layer` prop before searching right level ?

Reply:
I guess we should drop paths support and use `@bem/sdk.file.naming` instead

Reply:
Prob we shouldnt drop it because of levels paths.
→ https://github.com/bem/bem-sdk/issues/248